pym/bob/input.py

```diff
@@ -2731,6 +2736,12 @@ def generatePackages(self, nameFormatter, envOverrides={}, sandboxEnabled=False)
                 except ParseError as e:
                     e.pushFrame(root.getPackageName())
                     raise e
+            for i in self.__aliases:
+                try:
+                    p = walkPackagePath(result, self.__aliases[i])
+                    result[i] = p
+                except BuildError as e:
+                    print("Bad alias: " + str(e))
```

Comment:
colorize looks a bit strange,  only the 'Bad alias:' in in color 33. The build error is still in red. Maybe BobError should be extended to get the plain slogan?